qt_gui/src/qt_gui/about_handler.py

```diff
@@ -53,18 +53,16 @@ def __init__(self, qtgui_path, parent=None):
     def show(self):
         try:
             # append folder of 'qt_gui_cpp/lib' to module search path
-            qt_gui_cpp_path = os.path.realpath(get_package_path('qt_gui_cpp'))
+            qt_gui_cpp_path = os.path.realpath(get_package_path('qt_gui'))
```

Comment:
This change seems to contradict the comment. Can you please clarify why it was changed and either revert the change or update the comment accordingly.